domains/keli5.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "keli5",
+  "repo": "https://github.com/keli5/keli5.github.io",
+  "owner": {
+    "username": "keli5",
+    "email": ""
+  },
+  "record": {
+    "CNAME": "keli5.github.io"
```

Comment:
To force https on the website, have a look at https redirection in [the docs](https://github.com/is-a-dev/register/blob/main/API.md#record-required)

Reply:
My site doesn't support HTTPS and won't for a while yet.